### DISCUSSION TITLE
Duplicate database to another HOST

### DISCUSSION THREAD

**Discussion by DmitryGaluschkin** [Sequel-Ace/Sequel-Ace#1758] [Ideas]
Hi guys. 
I used the mysql sqlyog when I was in windows os in my view Sequel Ace much better than sqlyog. But I remember one function which I using quite often and liked it. It is a copy database from one host to another. 
How it is possible to add this functionality in future? I think it will be so comfortable to use this function, for example for easiest copy database from live server to localhost and vise versa. What do you think?

**Comment by Jason-Morcos** [comment]
Probably pretty tough to add this, but, if someone from the community wants to try, we'd be happy to review and merge that feature!
